mozilla_version/maven.py

```diff
@@ -54,3 +54,21 @@ def _compare(self, other):
             return -1
         else:
             return 0
+
+    @property
+    def is_beta(self):
+        """Maven versions can never be Beta."""
+        return False
+
+    @property
+    def is_release_candidate(self):
+        """Maven version can never be Release Candidates."""
+        return False
+
+    @property
+    def is_release(self):
```

Comment:
Good call on adding this one! I hadn't thought of it 🙂

Reply:
Should I be adding these with `attr.ib` instead?  Seems more concise and consistent with `is_snapshot`

Reply:
`is_release` does work, so of course, that would stay as I have it.

Reply:
That's a good question! 

As is, `attr.ib()` will change `__init__()` to allow `is_beta`/`is_release_candidate` as both positional and keyword arguments[1]. I think it may confuse `MavenVersion()` callers since this version class cannot be a beta or an RC. I like your approach with properties because it makes harder to allow `is_beta` to be true, for instance.

The good news is: we can tell `attr.ib()` we want this argument to be frozen! It's a new feature[2] of attrs v20.1.0. We can start to use it. If so, let's make the requirement clear at these 2 places:

https://github.com/mozilla-releng/mozilla-version/blob/e8fd64d4ce2058d8423bd16b799a6788e07855ee/requirements/base_py3.in#L1

https://github.com/mozilla-releng/mozilla-version/blob/e8fd64d4ce2058d8423bd16b799a6788e07855ee/requirements/base_py2.in#L1

[1] https://www.attrs.org/en/stable/api.html#attr.ib
[2] https://www.attrs.org/en/stable/api.html?highlight=on_setattr#setters